logdna/configs.py

```diff
@@ -3,5 +3,8 @@
     'MAX_LINE_LENGTH': 32000,
     'FLUSH_INTERVAL': 5,
     'FLUSH_BYTE_LIMIT': 2 * 1024 * 1024,
-    'LOGDNA_URL': 'https://logs.logdna.com/logs/ingest'
+    'LOGDNA_URL': 'https://logs.logdna.com/logs/ingest',
+    'BUFFER_BYTE_LIMIT': 4194304,
+    'BACKOFF_PERIOD': 8
```

Comment:
When I read "backoff" I think exponential backoff, and we're not doing that. Let's call it a retry timeout instead, and specify the units for clarity: `RETRY_INTERVAL_SECS`

Reply:
This is not quite a retry interval. It just take a longer time to send the next message, if the previous one was not delivered. For example, if the flush period is 5 it sends the request every 5 sec (unless buffer is empty, but if an exception was received, it will try to connect to the server in, sey, 10 sec, instead of 5.